dev/user.clj

```diff
@@ -0,0 +1,82 @@
+(ns dev.user
```

Comment:
Ok, that's fair. Well, should be easy to explain it to me then ;) Still feels like a candidate for a test